remotespark/utils/eventshandler.py

```diff
@@ -0,0 +1,13 @@
+from remotespark.utils.log import Log
+
+
+class EventsHandler:
+    def __init__(self):
+        self.logger = Log("EventsHandler")
+
+    def store_event(self, *args):
```

Comment:
I see your point but the main job for this method is just storing the event info, what do you think?